install.sh

```diff
@@ -380,6 +432,40 @@ initialize_icu_packages()
     fi
 }
 
+initialize_qrencode_packages()
+{
+    if [[ $OS == Darwin && !($BUILD_QRENCODE) ]]; then
+        # Update PKG_CONFIG_PATH for QRENCODE package installations on OSX.
+        # OSX provides libqrencodecore.dylib with no pkgconfig and doesn't support
+        # renaming or important features, so we can't use that.
+        HOMEBREW_QRENCODE_PKG_CONFIG="/usr/local/opt/qrencode/lib/pkgconfig"
+        MACPORTS_QRENCODE_PKG_CONFIG="/opt/local/lib/pkgconfig"
+
+        if [[ -d "$HOMEBREW_QRENCODE_PKG_CONFIG" ]]; then
+            export PKG_CONFIG_PATH="$PKG_CONFIG_PATH:$HOMEBREW_QRENCODE_PKG_CONFIG"
+        elif [[ -d "$MACPORTS_QRENCODE_PKG_CONFIG" ]]; then
+            export PKG_CONFIG_PATH="$PKG_CONFIG_PATH:$MACPORTS_QRENCODE_PKG_CONFIG"
+        fi
+    fi
+}
+
+initialize_libpng_packages()
+{
+    if [[ $OS == Darwin && !($BUILD_LIBPNG) ]]; then
+        # Update PKG_CONFIG_PATH for LIBPNG package installations on OSX.
+        # OSX provides liblibpngcore.dylib with no pkgconfig and doesn't support
+        # renaming or important features, so we can't use that.
+        HOMEBREW_LIBPNG_PKG_CONFIG="/usr/local/opt/libpng/lib/pkgconfig"
+        MACPORTS_LIBPNG_PKG_CONFIG="/opt/local/lib/pkgconfig"
+
+        if [[ -d "$HOMEBREW_LIBPNG_PKG_CONFIG" ]]; then
+            export PKG_CONFIG_PATH="$PKG_CONFIG_PATH:$HOMEBREW_LIBPNG_PKG_CONFIG"
+        elif [[ -d "$MACPORTS_LIBPNG_PKG_CONFIG" ]]; then
+            export PKG_CONFIG_PATH="$PKG_CONFIG_PATH:$MACPORTS_LIBPNG_PKG_CONFIG"
+        fi
+    fi
+}
+
```

Comment:
Yes, that was part of the caveat.  All bets are off there, as they are just cloned.